config

```diff
@@ -6,6 +6,13 @@ path /static StaticFileHandler {
   root static_files;
 }
 
+path /private StaticFileHandler {
+  root private_files;
```

Comment:
You should add "cp -r private_files deploy" to the deploy target in the Makefile